README.md

```diff
@@ -1,18 +1,57 @@
+
+### Responsiveness
+
+The React Toolbar component adapts to the available space with two types of display modes, scrollable and pop-up, for modern web and mobile applications.
+
+## Related links
+[Learn More about React Toolbar](https://www.syncfusion.com/react-components/react-toolbar?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/react?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/react/documentation/toolbar/getting-started?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-react-toolbar-component?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/react-js2?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/react?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+[Online example](https://ej2.syncfusion.com/react/demos/#/bootstrap5/toolbar/default?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples)
+
+
+## About Syncfusion React UI Components
+Syncfusion's [React UI Components](https://www.syncfusion.com/react-components?utm_source=github&utm_medium=listing&utm_campaign=react-tollbar-github-samples) library is the only suite that you will ever need to build an application since it contains over 80 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to Toolbar, we provide popular React Components such as [DataGrid](https://www.syncfusion.com/react-components/react-data-grid?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples), [Charts](https://www.syncfusion.com/react-components/react-charts?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples), [Scheduler](https://www.syncfusion.com/react-components/react-scheduler?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples), [Diagram](https://www.syncfusion.com/react-components/react-diagram?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples), and [Word Processor](https://www.syncfusion.com/react-components/react-word-processor?utm_source=github&utm_medium=listing&utm_campaign=react-toolbar-github-samples).
```

Comment:
spell error
<img width="737" alt="image" src="https://github.com/SyncfusionExamples/getting-started-with-the-react-toolbar-component/assets/61148953/c4d86d78-1d61-403a-be06-a0a56e25352f">